configure.ac

```diff
@@ -271,23 +259,86 @@ case $target in
 	  enable_pkey=yes
 	  ;;
      *s390*)
-	  CFLAGS="$CFLAGS -m31 -DNO_PKEY"
+	  CFLAGS="$CFLAGS -m31"
+		AC_DEFINE([NO_PKEY])
 	  ;;
      *ppc64* | *x86_64*)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
+		AC_DEFINE([NO_PKEY])
 	  ;;
      *ppc* | i*86*)
-	  CFLAGS="$CFLAGS -m32 -DNO_PKEY"
+	  CFLAGS="$CFLAGS -m32"
+		AC_DEFINE([NO_PKEY])
 	  ;;
      *)
-	  CFLAGS="$CFLAGS -DNO_PKEY"
+		AC_DEFINE([NO_PKEY])
 	  ;;
 esac
 AM_CONDITIONAL([NO_PKEY], [test "x$enable_pkey" != "xyes"])
 
+dnl Check and save whether the platform is AIX
+dnl Using the pthread library requires the _THREAD_SAFE macro to be set
+build_aix=no
+case "${host_os}" in
+    aix*)
+	     build_aix=yes
+			 AC_DEFINE([_ALL_SOURCE])
+			 AC_DEFINE([_THREAD_SAFE])
+			 with_systemd=no
+			 with_libica=no
+			 enable_ccatok=yes
+			 enable_swtok=yes
+			 enable_tpmtok=no
+			 enable_icsftok=no
```

Comment:
Why is the ICSF token disabled ? Its a remote token, and can generally operate on any platform. It just needs OpenLDAP to communicate with a remote ICSF server.

Reply:
If I remember correctly that needs libica, which AIX does not support. For the use case we were focusing on only involved the CCA token, so all efforts were concentrated on enabling that first.

Another problem is with the lack of equipment to test ICSF during the porting efforts.

Reply:
No it does not need libica.
Anyway, its up to you if you want to enable it or not. Either is fine with me.

Reply:
Ah, okay. I must be mixing it up with another token then.

I'll look into enabling support for it in the near future, always better to have as much parity as is possible across platforms.

Reply:
Yep, no hurry though. I do have a contact to POK who has an ICSF system to test with. Let me know when you need it for a test.